File: Source/Aladdin.CAPI.SCard.Java/src/aladdin/capi/scard/IAppletFile.java
package aladdin.capi.scard;
import java.io.*; 

///////////////////////////////////////////////////////////////////////////
// Файл апплета
///////////////////////////////////////////////////////////////////////////
public interface IAppletFile extends IAppletFileObject
{
    // прочитать данные выбранного файла
    void read(byte[] buffer, int offset) throws IOException;  

    // записать данные в выбранный файл
    void write(byte[] buffer, int offset) throws IOException;
}

File: Source/Aladdin.CAPI.SCard/IAppletFile.cs
namespace Aladdin.CAPI.SCard
{
    ///////////////////////////////////////////////////////////////////////////
    // Файл апплета
    ///////////////////////////////////////////////////////////////////////////
    public interface IAppletFile : IAppletFileObject
    {
        // прочитать данные выбранного файла
        void Read(byte[] buffer, int offset);  

        // записать данные в выбранный файл
        void Write(byte[] buffer, int offset);
    }
}
